index.html

```diff
@@ -5,38 +5,56 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-      rel="stylesheet"
-    />
     <title>Profile Rapheal Okonta</title>
   </head>
   <body>
     <header>
+     
       <nav class="nav_bar">
-        <a href="#" class="name_logo">RAPHEAL OKONTA</a>
-        <div class="link_header">
-        <div class="nav_header_link">
-          <a class="header_menu" href="#">Portfolio</a>
-          <a class="header_menu" href="#">About</a>
-          <a class="header_menu" href="#">Contact</a>
-          <img
+        <a href="#" class="nav_branding">RAPHEAL OKONTA</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#recent_works_link" class="nav_link">Portfolio</a>
+              <hr class="hr-line_break3">
+          </li>
+          <li class="nav_item">
+            <a href="#aboutmeProject" class="nav_link">About </a>
+          <hr class="hr-line_break3">
+                
+          </li>
+          <li class="nav_item">
+            <a href="#form_container_link" class="nav_link">contact</a>
+            <hr class="hr-line_break3">
+          </li>
```

Comment:
Kindly make sure that your design matches the figma design template as much as possible. The mobile menu is supposed to cover the whole screen when it appears (for all screen sizes of the mobile version) such that the elements behind it will not be visible. As shown in the image below, it is not doing so. Please rectify this anomaly. 

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 23-33-09](https://user-images.githubusercontent.com/65233860/196824369-7edf0327-4523-400e-b45f-153d75e8255b.png)  |  ![Screenshot from 2022-10-19 23-41-27](https://user-images.githubusercontent.com/65233860/196824662-46ddf1a0-3166-491f-82c1-bc9d124f113a.png)